libs/community/langchain_community/vectorstores/opensearch_vector_search.py

```diff
@@ -1043,6 +1227,34 @@ def _raw_similarity_search_with_score_by_vector(
                 vector_field,
                 score_threshold=score_threshold,
             )
+
+        elif search_type == HYBRID_SEARCH:
+            search_pipeline = kwargs.get("search_pipeline", "")
+            post_filter = kwargs.get("post_filter", {})
+            query_text = kwargs.get("query_text", "")
```

Comment:
is an empty `query_text` an actual possible path? if not, assert that query text exists in kwargs, and if it doesn't raise a helpful error for the user